terraform/modules/avl-producer-api/avl-feed-validator/main.tf

```diff
@@ -35,6 +35,15 @@ module "avl_feed_validator" {
         "arn:aws:ssm:${var.aws_region}:${var.aws_account_id}:parameter/subscription*"
       ]
     },
+    {
+      Action = [
+        "cloudwatch:PutMetricData",
+        "cloudwatch:GetMetricStatistics",
+        "cloudwatch:ListMetrics",
```

Comment:
what is the need for the `GetMetricStatistics` and `ListMetrics` permissions? Same question for the other terraform file changes

Reply:
I tried testing the changes with just PutMetricData however it was failing because of permission issue